i18n/fr/docusaurus-plugin-content-docs/version-22.04/administration/secure-platform.md

```diff
@@ -420,6 +462,21 @@ dnf install mod_ssl mod_security openssl
 yum install httpd24-mod_ssl httpd24-mod_security openssl
 ```
 
+</TabItem>
+
+<TabItem value="Debian 11" label="Debian 11">
+
+```shell
+sudo systemctl stop apache2
```

Comment:
After executing this block of command, i found an issue . We are supposed to modify the centreon.conf file later on in the documentation. After testing it twice, i found myself without the centreon.conf file at /etc/apache2/sites-available. Even when restarting with systemctl restart apache2, the file is not generated.

Reply:
This file is generated in the installation of the `centreon-web-apache` package (verify installation) in the `/etc/apache2/sites-available`  directory, and then a symbolic link is created from it to `/etc/apache2/sites-enabled`:

````
root@centreon-lab-1:/etc/apache2/sites-enabled# ls -lha
total 8.0K
drwxr-xr-x 2 root root 4.0K Aug 30 00:08 .
drwxr-xr-x 8 root root 4.0K Aug 30 00:08 ..
lrwxrwxrwx 1 root root   32 Aug 12 15:58 centreon.conf -> ../sites-available/centreon.conf
```